realm/realm-library/src/main/cpp/io_realm_internal_SharedRealm.cpp

```diff
@@ -57,6 +59,8 @@ JNIEXPORT void JNICALL Java_io_realm_internal_SharedRealm_nativeInit(JNIEnv* env
 {
     TR_ENTER()
 
+    hack_init();
```

Comment:
UH, yes, `JNI_OnLoad()` would be a better place! But first need to make this patch work ...